als_driver.c

```diff
@@ -0,0 +1,22 @@
+#define ALS_EN_GPIO 11
+#define ALS_READ_ADDR 0xA110FF30
+
+
+#include <stdio.h>
+#include <stdint.h>
+
+int als_read_reg(uint32_t als_reg_addr)
+{
+    printf("read value of 0x%X from als at addr 0x%X\n",
+           200, ALS_READ_ADDR);
+
```

Comment:
all you are doing is printing - what's the point?  You aren't really ready for a review, are you?

Reply:
This still isn't anything but a print - can you give me more information about the purpose of this pull request?

Reply:
I put those there as a placeholder so the rest of the team can fill in the rest.

Reply:
Ok, that makes sense